style.css

```diff
@@ -23,48 +25,140 @@ a {
   text-decoration: none;
 }
 
-header {
+/* header {
   position: fixed;
   top: 0%;
   z-index: 5;
   box-shadow: 0.1rem 0.1rem rgba(245, 243, 243, 0.918);
   background: #fff;
-  overflow: hidden;
-  background-color: #fff;
+} */
+
+.navbar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.right-nav {
+  color: black;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 40px;
+  position: fixed;
+  left: -100%;
+  top: 50px;
+  flex-direction: column;
+  background-color: #2f79da;
   width: 100%;
-  height: 916px;
+  text-align: center;
+  transition: 0.3s;
 }
 
-.header {
+/* .header {
   display: flex;
   width: 100%;
-  height: 90px;
+  height: 48px;
   padding: 14px 24px;
   align-items: center;
   justify-content: space-between;
-}
+}  */
 
-.header .my-logo {
+.my-logo {
   font-family: "Poppins", sans-serif;
   font-weight: 700;
   font-size: 18px;
-  color: #6070ff;
+  color: #d15e11;
 }
 
-.header .logo-link {
+.logo-link {
   text-decoration: none;
 }
 
-.right-nav {
+li {
+  list-style: none;
+}
+
+.nav-link:hover {
+  color: rgb(64, 169, 218);
+}
+
+.hamburger {
+  display: none;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: black;
+}
+
+@media (max-width: 500px) {
+  .hamburger {
+    display: block;
+  }
+
+  html,
+  body {
+    max-width: 100%;
+    overflow-x: hidden;
+  }
+
+  .right-nav {
+    display: flex;
+    width: 100%;
+  }
+
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+  }
+}
+
+/* .right-nav {
+  position: fixed;
+  left: -100%;
+  top: 70px;
+  gap: 0;
+  flex-direction: column;
+  background-color: #5ac057;
+  width: 400px;
+  text-align: center;
+  transition: 0.3s;
+} */
+
```

Comment:
- [ ] As you maybe noticed, most of your lines of code are commented out. I think this is the main reason for making your design dissimilar to the Figma template. Please, fix this issue to make sure that you are following all the required steps. If you have any blockage on working alone, you can reach out to student success to get more assistance.
**This is how looks your portfolio on `mobilepopup` branch.**
![image](https://user-images.githubusercontent.com/80612925/174879662-435ae476-ef05-4213-a00c-442b078924f8.png)